esphome/components/esp32_ble_tracker/automation.h

```diff
@@ -10,18 +10,22 @@ namespace esp32_ble_tracker {
 class ESPBTAdvertiseTrigger : public Trigger<const ESPBTDevice &>, public ESPBTDeviceListener {
  public:
   explicit ESPBTAdvertiseTrigger(ESP32BLETracker *parent) { parent->register_listener(this); }
-  void set_address(uint64_t address) { this->address_ = address; }
+  void set_address(const std::vector<uint64_t> &addresses) { this->address_vec_ = addresses; }
```

Comment:
Might as well rename the setter too...
```suggestion
  void set_addresses(const std::vector<uint64_t> &addresses) { this->address_vec_ = addresses; }
```

Reply:
Indeed I should have 👍 
It's fixed now.